static/js/dashboard.js

```diff
@@ -24,6 +21,10 @@ if (!Object.entries) {
 // requirement for dialog-polyfill
 import 'style!css!dialog-polyfill/dialog-polyfill.css';
 
+// material-ui requirement
+let injectTapEventPlugin = require("react-tap-event-plugin");
+injectTapEventPlugin();
```

Comment:
I'm not sure if it's required for AutoComplete but I wasn't able to click a SelectField without this code for some reason